src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -3907,6 +3910,7 @@ int writeFedAuthFeatureRequest(boolean write, /* if false just calculates the le
                             workflow = TDS.ADALWORKFLOW_ACTIVEDIRECTORYINTEGRATED;
                             break;
                         case ActiveDirectoryMSI:
+                        case ActiveDirectoryInteractive:
```

Comment:
Please create a separate enum for `ActiveDirectoryInteractive` to avoid confusion. See  `ADALWORKFLOW_ACTIVEDIRECTORYSERVICEPRINCIPAL` and `ADALWORKFLOW_ACTIVEDIRECTORYPASSWORD` for reference.